### ISSUE TITLE
Could not load NIB in bundle

### ISSUE THREAD

**Issue by nerthase** [iurichiba/SSMaterialCalendarPicker#1]
```
Terminating app due to uncaught exception 'NSInternalInconsistencyException', 
reason: 'Could not load NIB in bundle: '...' with name 'SSMaterialCalendarPicker''
```

The debugger is breaking at line `SSMaterialCalendarPicker.m` at line `86`. 
Following lines from `85`to `87`:

```
UINib *cellNib = [UINib nibWithNibName:kCalendarCellIdentifier bundle:nil];
self = [[[NSBundle mainBundle] loadNibNamed:kCalendarPickerIdentifier
                                      owner:self options:nil] objectAtIndex:0];
```

This is the content of my `Podfile`:

```
use_frameworks!

target 'MyProject' do

pod 'SSMaterialCalendarPicker', '~> 1.1'

end
```

**Comment by iurichiba** [comment]
The .xib file containing the picker view is missing, somehow. Thanks for reporting it.

I'll try to fix it this week, if I manage to free up some time; but if you find out first, please submit a Pull Request, it'll help a lot.

Also, I'm planning a big refactor of this pod, probably next month or so, it's quite slow as it is now.
Keep an eye out for a v2.0, if you decide to use another pod :)

**Comment by Mkey85** [comment]
Any updates here?
I'm still getting the same issue like nerthase

**Comment by grifas** [comment]
is there a solution ?

**Comment by iurichiba** [comment]
I tried to fix it on Friday, but I couldn't figure out how to fix it properly.
The problem is probably in the .pospec file, the .xib files are missing in the app's bundle.

I guess that, in the meantime, you can import it manually to your app's project. Just install it as usual, find the .xib files in the Pod folder, and import it to your project (and set its targets).

Sorry I didn't have the time to fix it. If you guys have any ideas, you're welcome to try, I'd gladly merge your pull requests :)

**Comment by iurichiba** [comment]
**GUYS!!1**

I **finally** fixed it, hahahaha! I'm sorry for the looong long wait, I've been busy with my own projects and work, but I finally got around to study this problem (aka try and try again, failing and failing until I got it) and fix it.

I don't know if you still want to use it, but if you do, give it a try :)

Thanks for the feedback, and I'll try to keep it stable. It's not the best calendar around, but I'm happy you guys were interested in the first place :)

Best regards,
Iuri Chiba.
